client/packages/system/src/Manage/Preferences/api/operations.graphql

```diff
@@ -9,6 +9,7 @@ query adminPreferenceList($storeId: String!, $prefType: PreferenceNodeType!) {
 query preferences($storeId: String!) {
   preferences(storeId: $storeId) {
     showContactTracing
+    displayPopulationBasedForecasting
   }
 }
```

Comment:
Same with this query, shouldn't exist anymore, just have the gql string in the usePreference hook